elodie/media/media.py

```diff
@@ -89,11 +89,20 @@ def get_coordinate(self, type='latitude'):
             if key not in exif:
                 continue
 
+            # Assign coordinate to variable
+            this_coordinate = exif[key]
+
+            # If exiftool GPS output is empty, the data returned will be a str
+            # with 0 length.
+            # https://github.com/jmathai/elodie/issues/354
+            if isinstance(this_coordinate, str) and len(this_coordinate) == 0:
+                continue
+
             # Cast coordinate to a float due to a bug in exiftool's
             #   -json output format.
             # https://github.com/jmathai/elodie/issues/171
             # http://u88.n24.queensu.ca/exiftool/forum/index.php/topic,7952.0.html  # noqa
-            this_coordinate = float(exif[key])
+            this_coordinate = float(this_coordinate)
```

Comment:
`float(exif[key])` per change above instead of assigning it and then casting later.